BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1099,3 +1099,5 @@
 "TextToShare" = "Text to share";
 "SendDetails" = "Send details";
 "SendNameRequired" = "Send name (required)";
+"CannotDeleteAccount" = "Cannot delete account";
+"CannotDeleteAccountDescription" = "This action cannot be completed because your account is owned by an organization. Contact your organization administrator for additional details.";
```

Comment:
⛏️ We usually use `DescriptionLong` as a suffix for these situations.
```suggestion
"CannotDeleteAccountDescriptionLong" = "This action cannot be completed because your account is owned by an organization. Contact your organization administrator for additional details.";
```